src/components/Tour/Tour.vue

```diff
@@ -98,80 +102,53 @@ export default {
     callbacks: {
       type: Object,
     },
+    steps: {
+      default: () => [],
+      type: Array,
+    },
   },
-    data () {
+  data () {
     return {
       tour: {
-        name:'app-list',
-        steps: [   
-          'app-list',
-          'low-code',
-          'crm',
-          'reporter',
-          'workflow',
-          'profile',
-          ].map(step=>{
-            return {
-                    name: step,
-                    target: `[data-v-onboarding="${step}"]`,
-                    header: {
-                      title: this.$t(`steps.${step}.title`),
-                    },
-                    content: this.$t(`steps.${step}.content`),
-                  }
-          }),
+        name: 'app-list',
```

Comment:
Shouldn't this be passed into the component, since it refers to corteza-webapp-one. 
That means when you implement tour for compose, you will have to change it here and webapp-one tour will stop working